dspace-api/src/main/java/org/dspace/uclouvain/itemEnhancer/enhancers/ItemEnhancerConfiguration.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.uclouvain.itemEnhancer.enhancers;
+
+import java.util.List;
+
+/**
+ * Configuration used by the metadata enhancer service {@link UCLouvainItemEnhancerService}.
+ * 
+ * @Author Michaël Pourbaix <michael.pourbaix@uclouvain.be>
+ */
+public class ItemEnhancerConfiguration {
+    private String sourceEntityType;
+    private List<String> sourceMetadataFields;
+    private String targetEntityType;
+    private List<String> targetMetadataFields;
+
+    // GETTERS && SETTERS
+
+    public void setSourceEntityType(String type) {
+        sourceEntityType = type;
+    }
+
+    public String getSourceEntityType() {
+        return sourceEntityType;
+    }
+
+    public void setSourceMetadataFields(List<String> fields) {
+        validateMetadataFields(fields);
+        sourceMetadataFields = fields;
+    }
+
+    public List<String> getSourceMetadataFields() {
+        return sourceMetadataFields;
+    }
+
+    public void setTargetEntityType(String type) {
+        targetEntityType = type;
+    }
+
+    public String getTargetEntityType() {
+        return targetEntityType;
+    }
+
+    public void setTargetMetadataFields(List<String> fields) {
+        validateMetadataFields(fields);
+        targetMetadataFields = fields;
+    }
+
+    public List<String> getTargetMetadataFields() {
+        return targetMetadataFields;
+    }
+
+    private void validateMetadataFields(List<String> fields) {
+        for (String field: fields) {
+            if (!isFieldValid(field)) {
+                throw new RuntimeException("Bad syntax for metadata field in metadata enhancer configuration."
+                    + " Should be of form '<schema>.<element>.<qualifier>'."
+                );
+            }
+        }
+    }
+
+    private boolean isFieldValid(String field) {
```

Comment:
Of interest to use `metadataFieldService.find()`  to know if the field exists in registries ?